src/views/components/download-results.html

```diff
@@ -47,9 +47,15 @@
         <input class='hidden govuk-checkboxes__input' name='type' type='checkbox' value='further-education-or-sixth-form-college-corporation' {{ selectedTypeCheckboxes.furtherEducationOrSixthFormCollegeCorporation }} aria-hidden='true' aria-labelledby='further-education-or-sixth-form-college-corporation'>
         <input class='hidden govuk-input govuk-input--width-10' name='dissolvedFrom' type='text' value='{{ dissolvedFrom }}' aria-hidden='true' aria-labelledby='dissolvedFrom'>
         <input class='hidden govuk-input govuk-input--width-10' name='dissolvedTo' type='text' value='{{ dissolvedTo }}' aria-hidden='true' aria-labelledby='dissolvedTo'>
+        {% if totalReturnedHits === 1 %}
```

Comment:
Check formatting here, did you run npm run lint ?

Reply:
Yes I did, and it was just the boolean one that had error.

Reply:
I will fix that now, thanks